bitforge/privkey.py

```diff
@@ -1,126 +1,178 @@
-import random, struct, binascii, collections
-import network, utils
-from errors import *
-from network import Network
-from pubkey import PublicKey
-from address import Address
-from encoding import *
+import collections
 
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import ec
 
-rng     = random.SystemRandom()
-KEY_MAX = utils.generator_secp256k1.order()
+from bitforge import encoding, error, network, pubkey, tools
 
-def random_secret():
-    return rng.randint(1, KEY_MAX - 1)
 
+class Error(error.BitforgeError):
```

Comment:
I did a little bit of research on this, and it seems like the common practice is to use the module-level namespacing instead of classes, I'm in the process of taking errors out of the classes